runner.py

```diff
@@ -98,10 +97,14 @@ def parse_args():
         args.target = config.ALL_TARGETS
 
     if not args.serial:
-        args.serial = config.DEVICE_SERIAL_PORT
+        # no support for running 2 device targets at once using one host
```

Comment:
Because of using same sockets (and maybe some other issues) we can't now connect two boards to host and start runner for them. I am not sure, whether is worth to support such utility. Please let me know what do You think about it.